### PR TITLE
Created div/columns to hold tables

### PR DESCRIPTION
**Pull request by IsaacPeters**
Used 3 divs, 1 row which holds 2 columns, each of which contain a battleGrid table

### CODE REVIEW

File: src/main/java/views/ApplicationController/index.ftl.html [hunk 1]
```diff
@@ -4,12 +4,16 @@
...
-<table id="opponent" class="battleGrid">
-</table>
-
-<table id="player" class="battleGrid">
-</table>
-
+<div class="row">
+    <div class="column" style="display: inline-block">
+        <table id="opponent" class="battleGrid"></table>
+        </table>
```
Comment (Hartwelg) [inline src/main/java/views/ApplicationController/index.ftl.html]: why do you have </table> here when it exists on the line above already?

Comment (Hartwelg) [reply src/main/java/views/ApplicationController/index.ftl.html]: *</table>

Comment (IsaacPeters) [reply src/main/java/views/ApplicationController/index.ftl.html]: This has been fixed

**Comment by IsaacPeters** [comment]
Accidentally got closed, going to go ahead and merge it now

**Pull request merged**